### PR TITLE
fix: [CDS-47204]: Hide Active deployments for Gitops services

### PR DESCRIPTION
**Pull request by cjlee01**
### Summary

hotfix for Hide Active deployments for Gitops services
https://github.com/harness/harness-core-ui/pull/12740

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstances.tsx [hunk 1]
```diff
@@ -97,11 +108,15 @@ export const ActiveServiceInstances: React.FC = () => {
               </>
             }
           />
-          <Tab
-            id={ServiceDetailTabs.DEPLOYMENT}
-            title={getString('cd.serviceDashboard.recentDeployments')}
-            panel={<Deployments />}
-          />
+          {gitopsEnabled ? (
+            <></>
```
Comment (GeorgiosP) [inline src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstances.tsx:112]: Hide the tabs when GitOps is enabled right?

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesV2.tsx [hunk 2]
```diff
@@ -67,6 +78,7 @@ export const ActiveServiceInstancesV2: React.FC = () => {
     return Boolean(
       activeInstanceData &&
         deploymentData &&
+        !gitopsEnabled &&
```
Comment (GeorgiosP) [inline src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesV2.tsx:81]: This is becoming an overloaded Boolean value, we should be careful when doing this.

**Review by GeorgiosP — APPROVED** [review]
Just some observations not immediate changes needed

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstances.tsx (hunk 1 above)
Comment (cjlee01) [reply src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstances.tsx:112]: correct @GeorgiosP

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesV2.tsx (hunk 2 above)
Comment (cjlee01) [reply src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesV2.tsx:81]: noted

**Review by ishantmehtaharness — APPROVED** [review]

**Review by tan-nhu — APPROVED** [review]

**Pull request merged**